easybuild/easyconfigs/n/NVHPC/NVHPC-20.7-GCC-9.3.0.eb

```diff
@@ -0,0 +1,43 @@
+name = 'NVHPC'
+version = '20.7'
+local_gccver = '9.3.0'
+versionsuffix = '-GCC-%s' % local_gccver
+
+homepage = 'https://developer.nvidia.com/hpc-sdk/'
+description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)"""
+
+toolchain = SYSTEM
+
+sources = ['nvhpc_2020_%(version_major)s%(version_minor)s_Linux_x86_64_cuda_multi.tar.gz']
```

Comment:
No public download for this?
Can we use the `%(arch)s` template rather than hardcoding `x86_64`?

Also need a SHA256 checksum

Reply:
This is just a public download, so please add:

```python
source_urls = ['https://developer.download.nvidia.com/hpc-sdk/%(version)s/']
sources = ['nvhpc_2020_%(version_major)s%(version_minor)s_Linux_x86_64_cuda_multi.tar.gz']
checksums = ['a5c5c8726d2210f2310a852c6d6e03c9ef8c75e3643e9c94e24909f5e9c2ea7a']

Reply:
Ah, that has changed. But one still needs to accept the Software License Agreement. Are you sure we can just do that?

Reply:
(I've already added the SHA-sum but not yet uploaded the new file. It's a bit tricky to handle this abstract file and the specific JSC file.)

Reply:
I think it's fine to add the source URL and auto-download it, since the download URL is public.

Please use a SHA256 checksum (use `eb --inject-checksums` for example, or the `sha256sum` command on Linux).

Reply:
Really? The download URL can only be seen after confirming to the License Agreement. We circumvent this check if we just download it.

I'll update the checksum to SHA256.

Reply:
On one hand, nvidia's fault for not protecting the tarball better. On the other hand, we do know they want us to accept the license agreement. So maybe we should not have auto-download 🤷

Reply:
I'm fine with not making this auto-download, but we should add a comment above `sources` with some simple instructions where to download, and mention "requires accepting license agreement".

Reply:
Added!
Should I include something to the likes of the following:
```
# If you accept the License Agreement, you may also uncomment the following line to automatically download the sources
# source_urls = ['https://developer.download.nvidia.com/hpc-sdk/%(version)s/']
sources = ['nvhpc_2020_%(version_major)s%(version_minor)s_Linux_x86_64_cuda_multi.tar.gz']
checksums = ['a5c5c8726d2210f2310a852c6d6e03c9ef8c75e3643e9c94e24909f5e9c2ea7a']
```

Reply:
@AndiH That makes a lot of sense to me, yes.

Reply:
Added in 485bf1d622!